napari/layers/labels/labels.py

```diff
@@ -1588,6 +1514,13 @@ def _get_shape_and_dims_to_paint(self) -> Tuple[list, list]:
     def _get_dims_to_paint(self) -> list:
         return list(self._slice_input.order[-self.n_edit_dimensions :])
 
+    def _get_pt_not_disp(self, indices, values):
+        slice_input = self._slice.slice_input
+        point = np.round(
+            self.world_to_data(slice_input.world_slice.point)
+        ).astype(int)
+        return {dim: point[dim] for dim in slice_input.not_displayed}
```

Comment:
This is weird... indices and values aren't used? Can you add a docstring to help me understand why the function signature is what it is?

Reply:
Yes. I'm sorry. It is an artifact of simplification of code. First version was using it.